test/server/badge.routes.test.js

```diff
@@ -105,7 +105,7 @@ describe('badge.routes.test.js', () => {
       async () => {
         const resText = await fetchText('/apex/tiers/sponsors/0/avatar.svg?isActive=false');
         expect(resText).toMatch(
-          /<svg xmlns="http:\/\/www.w3.org\/2000\/svg" xmlns:xlink="http:\/\/www.w3.org\/1999\/xlink" width="64" height="64">/,
+          /<svg xmlns="http:\/\/www.w3.org\/2000\/svg" xmlns:xlink="http:\/\/www.w3.org\/1999\/xlink"/,
```

Comment:
Unrelated change (it's crashing in other PRs), but this will improve the reliability of this test since the returned result can either be a user avatar or an organization logo and they seem to have different sizes.